postgres/experiment.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: redskyops.dev/v1alpha1
 kind: Experiment
 metadata:
-  name: postgres-example-test
+  name: postgres-example
```

Comment:
I'm fine with this change, but it'll also require doc updates:
https://raw.githubusercontent.com/redskyops/redskyops-controller/master/docs/quickstart.md